src/widgets/refinement-list/refinement-list.js

```diff
@@ -66,7 +66,7 @@ const renderer = ({
       showMore={showMoreConfig !== null}
       templateProps={renderState.templateProps}
       toggleRefinement={refine}
-      searchFacetValues={searchForFacetValues && searchForItems}
+      searchFacetValues={searchForFacetValues ? searchForItems : () => {}}
```

Comment:
I find it overkill and not beautiful to use the spread operator to achieve this, I can do either:

```jsx
searchFacetValues={searchForFacetValues ? searchForItems : () => {}}
```

or 

```jsx
searchFacetValues={searchForFacetValues ? searchForItems : undefined}
```